mirage/scenarios/registrations.ts

```diff
@@ -54,6 +54,15 @@ export function registrationScenario(
 
     server.create('registration', { id: 'beefs' });
 
+    server.create('resource', {
+        pid: '2',
+        name: 'Code',
+        description: 'Code Badge',
+        dateCreated: new Date(),
+        dateModified: new Date(),
+        finalized: true,
+    });
+
```

Comment:
This doesn't look like it's going to be attached to any registration, so you might not need this.